### ISSUE TITLE
Can't get Mica effect working on W11 22621.1344

### ISSUE THREAD

**Issue by eramdam** [QNetITQ/WaveFox#117]
I'm running
Edition	Windows 11 Pro
Version	22H2
Installed on	‎11/‎23/‎2022
OS build	22621.1344
Experience	Windows Feature Experience Pack 1000.22639.1000.0

Alongside the newest WaveFox release and I have MicaForEveryone setup with the following options and I'm not getting any transparency on the Firefox window. Am I missing something?

![image](https://user-images.githubusercontent.com/1409924/222928291-87129d43-1e37-46dc-8c9b-9e26221b0b0f.png)
![image](https://user-images.githubusercontent.com/1409924/222928303-877dab3f-4646-436b-943f-81d1d7348a5b.png)
![image](https://user-images.githubusercontent.com/1409924/222928310-99638d08-cb9c-4b9f-a792-31168b5dfd49.png)

**Comment by QNetITQ** [comment]
The accent color should be turned off.

**Comment by eramdam** [comment]
Oh! Thanks, that seems to fix it! The contrast of the tabs/toolbar seems a bit low compared to the screenshots, even with the transparency disabled though 
![image](https://user-images.githubusercontent.com/1409924/222932695-a29cdc1d-0f1f-493d-ae79-f6f22c0286ea.png)

**Comment by dddictionary** [comment]
where can i change accent color?

![image](https://user-images.githubusercontent.com/86024593/230784515-98f7aa96-e2b5-4044-b18f-c49afe986869.png)

![image](https://user-images.githubusercontent.com/86024593/230784520-3f0aa3fd-4dcf-4766-af5a-0798472e2fd2.png)


This is what i have right now. I just searched accent colors in the windows search bar.

I followed the above images and created a new rule for firefox with all settings at default in the Mica GUI.


My userChrome settings:
![image](https://user-images.githubusercontent.com/86024593/230784694-c2bc0fcc-0c15-4f45-ab0b-234341beee71.png)
